infra/gp-multena/templates/deployment.yaml

```diff
@@ -87,6 +95,16 @@ spec:
           secret:
             secretName: {{ include "gp-multena.secretName" . }}
           {{- end }}
+          {{- if eq .Values.tlsCopy.loki.enabled true }}
```

Comment:
bin mir nicht sicher, ob wir das mounting des secrets an das kopieren koppeln wollen. 
kann ja sein, dass der proxy verwendet werden möchte, ohne kyverno am cluster zu haben und die:der anwender:in das zertifikat manuell überspielt, oder?